server/routes/middleware/Token.js

```diff
@@ -4,52 +4,58 @@ const User = require('../../models/User');
 var privateKey = "mySecret";
 
 var verifyToken = function (req, res, next) {
-  console.log("Verifying token.");
-  var token = req.body.token || req.query.token || req.headers['x-access-token'] || req.headers['authorization'].split(' ')[1]; // normal headers "Authorization: Bearer 2kj234df0ds2f3n40n"
```

Comment:
This OR statement handles all different ways the token is passed.

Reply:
But yes, this fails when no token is provided. We shouldn't pass headers with the authorization tag anyways. Will fix after front end changes it to x-access-token.

Reply:
No the OR statement is not handling that for some reason, I tried all different ways but all were giving the error that it cannot split.